src/DIRAC/WorkloadManagementSystem/Client/JobState/CachedJobState.py

```diff
@@ -106,7 +106,7 @@ def commitChanges(self):
 
     def serialize(self):
         if self.__manifest:
-            manifest = [self.__manifest.dumpAsCFG(), self.__manifest.isDirty()]
+            manifest = ["".join(self.__manifest.dumpAsJDL().split()), self.__manifest.isDirty()]
```

Comment:
Why dumping as JDL now, when it was previously dumped as CFG?

Reply:
Because now when the manifest is deserialized i'm now using the load method that looks for the jdl format. My goal is to remove loadJDLAsCFG and  dumpCFGAsJDL parsers in favor of the ClassAd one.

Reply:
I didn't intend for this change to be in this PR, i will remove it.